src/types/onyx/CancellationDetails.ts

```diff
@@ -0,0 +1,29 @@
+import type {CancellationType, FeedbackSurveyOptionID} from '@src/CONST';
+import type * as OnyxCommon from './OnyxCommon';
+import type PrivateSubscription from './PrivateSubscription';
+
+/** Cancellation details model */
+type CancellationDetails = {
+    /** Cancellation date */
+    cancellationDate?: string;
+
+    /** Cancellation reason */
+    cancellationReason?: FeedbackSurveyOptionID;
```

Comment:
I don't have context but can this be optional?

Reply:
It has to be so the API call in `Subscription.ts` works correctly, I can try to refactor that if you think it shouldn't be optional

Reply:
I don't think this should be optional. Lets see if we can refactor 👍.